docker/Dockerfile

```diff
@@ -136,3 +134,56 @@ EXPOSE 1935
 ENTRYPOINT ["/init"]
 
 CMD ["python3", "-u", "-m", "frigate"]
+
+
+# Frigate with Node.js and NPM
+FROM frigate-without-web AS frigate-with-node
+
+# Install Node 16
+RUN apt-get update -y \
+    && apt-get install -y curl \
+    && curl -sL https://deb.nodesource.com/setup_16.x | bash - \
+    && apt-get install -y nodejs
+
+RUN npm install -g npm@latest
+
+
+# Build of the Frigate web
+FROM frigate-with-node AS web-build
+
+WORKDIR /work
+COPY web/package.json web/package-lock.json ./
+RUN npm install
+
+COPY web/ ./
+RUN npm run build
+
+
+# Frigate Container
+FROM frigate-without-web
+
+COPY --from=build-web /work/dist web/
```

Comment:
We also want the build output copied for the devcontainer. Often times, I am only working on backend features and don't want to run the dev server for the frontend.

Reply:
I'm planning to solve that by setting creating proper VS Code tasks and debug configuration launch that would build the UI when pressing F5.

Reply:
Addressed in https://github.com/blakeblackshear/frigate/pull/3492/commits/f4b2c3f37121e61ae113180e3a85827dc72f555c